lib/main.ts

```diff
@@ -259,6 +259,10 @@ export class Transpiler {
   }
 
   private normalizeSlashes(path: string) { return path.replace(/\\/g, '/'); }
+  
+  private translateComment(comment: string): string {
+    return comment.replace(/\{@link ([^\}]+)\}/g, '[$1]');
```

Comment:
Should the inner group `[^\}]+` be non-greedy here? Why does this work for the two-link test case below?